pfb-analysis/scripts/entrypoint.sh

```diff
@@ -38,6 +38,20 @@ do
     ((counter++))
 done
 
+PFB_TEMPDIR=`mktemp -d`
+
+# If given an S3 key for the shapefile, dowload and unzip it. Overrides PFB_SHPFILE.
+if [ "${PFB_SHPFILE_KEY}" ]
```

Comment:
Yeah, wasn't sure which would be preferable, so I did the one then thought the expiring URLs might be a bother and switched to the other.  I'll switch back.